appointment/upload-documents/uploadDocumentsDataService.js

```diff
@@ -69,6 +69,24 @@ define('uploadDocumentsDataService', [], function($http) {
 				}, (error) => {
 					return null;
 				});
+			},
+			storeConsent: function(reviewConsent, virtualConsent, signature, appointmentId) {
+				return $http({
+					url: '/server/appointment/upload-documents/uploadDocuments.php',
+					method: 'POST',
+					data: 'action=storeConsent' +
+						`&reviewConsent=${reviewConsent}` +
+						`&virtualConsent=${virtualConsent}`+
+						`&signature=${signature}`+
```

Comment:
What happens if someone has a space in their signature? This still works, right? It should, but worth testing

Reply:
Still works!